src/borda_clj/client.clj

```diff
@@ -45,28 +44,37 @@
     (let [[v ch] (alts! [report-ch [measurements-ch measurements]])]
       (cond
         (= ch measurements-ch) (recur {})
-        (nil? v) (close! kill-ch)
+        (nil? v) (do (close! kill-ch) (>! measurements-ch measurements))
```

Comment:
@oxtoacart this fix from this branch didn't make it into master yet.

Reply:
What does this do again?

Reply:
Ah, gotcha.